auto_nag/scripts/stepstoreproduce.py

```diff
@@ -0,0 +1,73 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this file,
+# You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from auto_nag.bugbug_utils import BugbugScript
+from auto_nag.utils import nice_round
+from bugbug.models.stepstoreproduce import StepsToReproduceModel
+
+
+class StepsToReproduce(BugbugScript):
+    def __init__(self):
+        self.model_class = StepsToReproduceModel
+        super().__init__()
+
+    def description(self):
+        return '[Using ML] Bugs with missing steps to reproduce'
+
+    def columns(self):
+        return ['id', 'summary', 'has_str', 'confidence', 'autofixed']
+
+    def sort_columns(self):
+        return lambda p: (-p[3], -int(p[0]))
+
+    def get_bz_params(self, date):
+        start_date, end_date = self.get_dates(date)
+
+        reporter_skiplist = self.get_config('reporter_skiplist', default=[])
+        reporter_skiplist = ','.join(reporter_skiplist)
+
+        params = {
+            'bug_type': 'defect',
+            'f1': 'longdesc', 'o1': 'changedafter', 'v1': start_date,
+            'f2': 'longdesc', 'o2': 'changedbefore', 'v2': end_date,
+            'f3': 'reporter', 'o3': 'nowords', 'v3': reporter_skiplist,
+            'f4': 'cf_has_str', 'o4': 'equals', 'v4': '---',
+            'n5': 1, 'f5': 'cf_has_str', 'o5': 'changedafter', 'v5': '1970-01-01',
+            # Ignore closed bugs.
+            'bug_status': '__open__',
+        }
+
+        return params
+
+    # TODO: Remove this filtering once this is fixed in bugbug.
```

Comment:
This will be fixed in next bugbug's release: https://github.com/mozilla/bugbug/commit/4f77ac82e36090cd657be5ce1f7f2f30e0a9da16.